config/application.rb

```diff
@@ -23,5 +23,8 @@ class Application < Rails::Application
     #
     # config.time_zone = "Central Time (US & Canada)"
     # config.eager_load_paths << Rails.root.join("extras")
+
+    # Use the Postgresql-specific syntax for DB dumps
+    config.active_record.schema_format = :sql
```

Comment:
change the default format for schema dumps to support postgresql-specific features

Reply:
```
PG::UndefinedTable: ERROR:  relation "challenge_owners_id_seq" does not exist (ActiveRecord::StatementInvalid)
```

I was investigating this local schema error when running `bin/rails db:prepare` and found this [Rails guide article](https://edgeguides.rubyonrails.org/active_record_migrations.html#types-of-schema-dumps) which recommends to use the `:sql` dumper instead of `:ruby` schema_format. this will let us dump & load postgresql-specific features like the `nextva()` function which is used for bigint ID sequences.